### PR TITLE
Default to generate both server and client

### PR DESCRIPTION
**Pull request by johanandren**
Simplified sbt plugin config
and some more docs

Refs #159

### CODE REVIEW

File: docs/src/main/paradox/sbt-plugin.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+It will then by default look for `.proto` files under `src/main/protobuf`.
+
+## Configuration
+
+By default the plugin generates both a client and a server stub for Scala. 
+
+It can be configured to just generate either server or client like so:
+
+```scala
```
Comment (raboof) [inline docs/src/main/paradox/sbt-plugin.md]: Should we use Scala/Java tabs here?

File: docs/src/main/paradox/index.md [hunk 2]
```diff
@@ -23,5 +23,6 @@ Based on a protobuf service definition, akka-grpc can generate:
 
 * [server](server.md)
 * [client](client.md)
+* [sbt plugin details](sbt-plugin.md)
```
Comment (raboof) [inline docs/src/main/paradox/index.md]: 'sbt configuration'? but :+1: as discussed on slack

File: sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala [hunk 3]
```diff
@@ -30,12 +30,15 @@ object AkkaGrpcPlugin extends AutoPlugin {
   }
   import autoImport._
 
-  override def projectSettings = configSettings(Compile) ++ configSettings(Test)
+  override def projectSettings = defaultSettings ++ configSettings(Compile) ++ configSettings(Test)
+
+  private def defaultSettings =
+    Seq(
+      akkaGrpcCodeGeneratorSettings := Seq("flat_package"),
+      akkaGrpcCodeGenerators := GeneratorAndSettings(ScalaBothCodeGenerator, akkaGrpcCodeGeneratorSettings.value) :: Nil)
```
Comment (raboof) [inline sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala]: :+1:

File: docs/src/main/paradox/sbt-plugin.md (hunk 1 above)
Comment (johanandren) [reply docs/src/main/paradox/sbt-plugin.md]: You mean for the generate-scala/java sources? I thought about that when writing it and that this is more for completeness in a complicated setup than two alternatives as it normally is with Java/Scala

File: sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala (hunk 3 above)
Comment (johanandren) [reply sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala]: Couldn't move the third setting out of the config-scope for some reason, but couldn't figure out why

**Pull request merged**